src/vue/backend/src/components/APIKey.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <idx-form-group class="idx-wp-needs-validation" :class="{'idx-wp-was-validated': success|error}" novalidate>
```

Comment:
Two instances of `class` are used. Use `customClass` instead and combine the two. Also, remove the `idx-wp-` to avoid double prefixing